core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2669,6 +2547,15 @@ public String toString() {
               + "aggregated, so different applications must set their own ids or leave this value "
               + "unset to use a randomly generated id.")
           .build();
+  public static final PropertyKey USER_NETWORK_DATA_TIMEOUT_MS =
```

Comment:
how is this different from `USER_NETWORK_KEEPALIVE_TIMEOUT_MS` below`?

Reply:
The `USER_NETWORK_KEEPALIVE_TIMEOUT_MS ` is used by gRPC channel to determine how long it should wait for a response for a keepalive ping before closing the connection. This is for detecting a dead server.

`USER_NETWORK_DATA_TIMEOUT_MS` is the maximum wait time for Alluxio client to wait for a stream response before giving up and throw an exception. This type of timeout can be triggered even a server is perfectly responsive. E.g. if the reader is trying to read a block of data from worker that took too long to be loaded from UFS.